.github/workflows/unit-test.yml

```diff
@@ -52,8 +52,9 @@ jobs:
           fi
           npm run test
       - name: Report Coverage
-        run: npm run codecov
-        if: ${{ matrix.node_version == '14' }}
```

Comment:
so now we're collecting coverage for all node versions?

Reply:
yep :slightly_smiling_face: 
I figured that's better anyway as there we actually won't end up with supposedly-missing where we do different things for different Node.js versions.

Reply:
Works better here than in contrib, where we sometimes don't run all the tests to save time. 
Here we always run the full suite of tests on both PRs and `main`, so we won't have any inconsistencies by doing this :slightly_smiling_face: